13-rticles.Rmd

```diff
@@ -99,22 +99,22 @@ bibliography: bibliography.bib
 output: rticles::springer_article
 ```
 
-As the Rmd documents are built using customized templates, you may not be able to use the YAML metadata to control the layout of the document as described in Section \@ref(pdf-document), unless the template supports such metadata. For example, adding `toc: true` may not add a table of contents. Commands that control the building process may still be used though, including `keep_tex: true`, or those that configure **knitr** chunk options (e.g., `fig_width`).
+As the Rmd documents are built using customized templates, you may not be able to use the YAML metadata\index{YAML metadata} to control the layout of the document as described in Section \@ref(pdf-document), unless the template supports such metadata. For example, adding `toc: true` may not add a table of contents. Commands that control the building process may still be used though, including `keep_tex: true`, or those that configure **knitr** chunk options (e.g., `fig_width`)\index{knitr chunk options}.
 
 ## LaTeX content {#rticles-latex}
 
-As the only output format of the **rticles** formats is PDF, the content of the documents may include raw LaTeX formatting. This means you may use LaTeX to produce figures and tables (if you have to), e.g.,
+As the only output format of the **rticles** formats is PDF, the content of the documents may include raw LaTeX formatting\index{LaTeX formatting}. This means you may use LaTeX to produce figures and tables (if you have to), e.g.,
 
 ```latex
 \begin{figure}[ht]
 \centering
 \includegraphics[width=\linewidth]{foo}
 \caption{An example image.}
 \label{fig:foo}
-\end{figure} 
+\end{figure}
 ```
 
-Unless you have specific requirements for using LaTeX, we recommend that you use the R Markdown syntax. This keeps you work generally more readable (in terms of the source document), and less prone to formatting errors. For example, the above code block would be better represented as:
+Unless you have specific requirements for using LaTeX, we recommend that you use the R Markdown syntax\index{R Markdown syntax}. This keeps you work generally more readable (in terms of the source document), and less prone to formatting errors. For example, the above code block would be better represented as:
```

Comment:
An index is for helping readers navigate to the relevant page by keywords, so the keyword should be the key topic that a particular page or paragraph is talking about. For this paragraph, it does not actually talk about the "R Markdown syntax", so if a reader comes here from the keyword, he might be confused.

Reply:
I will edit that. Thank you.